python_on_whales/exceptions.py

```diff
@@ -9,20 +9,23 @@ def __init__(
         stdout: Optional[bytes] = None,
         stderr: Optional[bytes] = None,
     ):
-        command_launched_str = " ".join(command_launched)
+        self.docker_command = " ".join(command_launched)
+        self.return_code = return_code
+        self.stderr = stderr.decode() if stderr else ""
+        self.stdout = stdout.decode() if stdout else ""
         error_msg = (
-            f"The docker command executed was `{command_launched_str}`.\n"
+            f"The docker command executed was `{self.docker_command}`.\n"
             f"It returned with code {return_code}\n"
         )
         if stdout is not None:
-            error_msg += f"The content of stdout is '{stdout.decode()}'\n"
+            error_msg += f"The content of stdout is '{self.stdout}'\n"
         else:
             error_msg += (
                 "The content of stdout can be found above the "
                 "stacktrace (it wasn't captured).\n"
             )
         if stderr is not None:
-            error_msg += f"The content of stderr is '{stderr.decode()}'\n"
+            error_msg += f"The content of stderr is '{self.stderr}'\n"
```

Comment:
Can you provide a small user guide explaining the attributes available? 
Just add a markdown file here: https://github.com/gabrieldemarmiesse/python-on-whales/tree/master/docs/template/user_guide

and declare it here: https://github.com/gabrieldemarmiesse/python-on-whales/blob/master/docs/mkdocs.yml

thanks!

Reply:
Done in last version or PR (c810682).